### PR TITLE
Increase "Back to View Work Order Details" font, size, visibility

### LINKED ISSUE #150: Increase "Back to View Work Order Details" font, size, visibility
As a technician who is on the Location Details page. When they get done adding assets, they don't know to go back to the Work Order Details page. The button is currently small. It's below the Asset Form. 

https://atd.knack.com/24-july-2019--test-signs-and-markings-operations#my-work-orders/view-work-orders-details-sign/5d39d5bfbf7f32001058dd7a/view-work-order-signs-location-details
/5d39f05e3b39c70010b72cc1/

### PR DESCRIPTION
**Pull request by mddilley**
Closes #150

### CODE REVIEW

**Comment by mddilley** [comment]
This PR increases the font size and weight of the "Back to View Work Order Details" link on the Location Details view.

@dianamartin I ended up moving the back link to the top of the view, but I then realized that after adding an asset the page stays scrolled to the bottom where the link originally lived. Do you think it makes more sense to have the back link at the top of the page or the bottom of the page? Here are some screenshots.

*Link at top*
Screen Shot 2019-07-26 at 11 19 25 AM

*Link at bottom*
Screen Shot 2019-07-26 at 11 19 08 AM

**Comment by dianamartin** [comment]
@mddilley I think we should keep it on the bottom where it is because that's likely where the user will be after they submitted an asset. 

Maybe we should put an icon too? 
https://fontawesome.com/v4.7.0/icons/
image

I was thinking the `arrow-left` or `arrow-circle-left`. What do you think?

**Comment by mddilley** [comment]
@dianamartin Sounds great! I love icons. I'll move that link back to its home at the bottom and add an arrow in.

**Comment by mddilley** [comment]
@dianamartin Here is the updated link with the circle arrow from Font Awesome.

Screen Shot 2019-07-26 at 12 51 20 PM

**Comment by dianamartin** [comment]
@mddilley Kati and I checked it out and we think it looks great!

**Comment by dianamartin** [comment]
@mddilley Did we do this for the Signs and Markings PROD environment and not just TEST correct?

**Comment by mddilley** [comment]
@dianamartin Yes, this is now in PROD and TEST.

**Comment by mddilley** [comment]
@mateoclarke Most of the changes in this PR are Prettier formatting. The two non-formatting changes are reverting the iFrame's src back to the main Netlify link (it was pointing to a preview link) and also lines 104-117 in /knack/index.css.

File: knack/index.css [hunk 1]
```diff
@@ -85,6 +97,21 @@ a.big-button {
...
+  cursor: pointer;
+}
+
+/* Make back link in Location Details View bold, larger, and add FontAwesome back arrow */
+#kn-scene_1039 > div.kn-view.kn-back-link > a {
+  font-weight: 700;
+  font-size: 18px;
+}
+
+#kn-scene_1039 > div.kn-view.kn-back-link > a:before {
```
Comment (mateoclarke) [inline knack/index.css]: 👍

**Pull request merged**